lib/imperative-test.js

```diff
@@ -447,6 +447,31 @@ class ImperativeTest extends Test {
     });
   }
 
+  resolvesTest(input, expected, check) {
+    return this.test('resolves test', async t => {
+      if (typeof input === 'function') input = input();
+      const res = await input;
+      t.strictSame(res, expected);
+      if (check) check(t, res);
+      t.end();
+    });
+  }
+
+  rejectsTest(input, err, check) {
+    return this.test('rejects test', async t => {
+      if (typeof input === 'function') input = input();
+      try {
+        t.strictSame(await input, undefined, 'must not resolve');
```

Comment:
I'd prefer to omit this `strictSame`.

Reply:
I will change this to be stored in `fail` result.